hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
I have the same question as clayton re: downside.  Static would be good unless we want to rethink how the images of this test are going to be structured.